2025-03-09 21:59:00,265 INFO    Thread-16 (_run_job):8340 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_215900-glm3byz1\logs\debug.log
2025-03-09 21:59:00,266 INFO    Thread-16 (_run_job):8340 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_215900-glm3byz1\logs\debug-internal.log
2025-03-09 21:59:00,266 INFO    Thread-16 (_run_job):8340 [wandb_init.py:init():761] calling init triggers
2025-03-09 21:59:00,266 INFO    Thread-16 (_run_job):8340 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'relu', 'batchSize': 64, 'hiddenLayer_Size': 32, 'learningRate': 0.001, 'num_epochs': 10, 'num_hiddenLayers': 5, 'optimizer': 'adam', 'weightDecay': 0, 'weightInit': 'random'}
config: {'num_epochs': 10, 'num_hiddenLayers': 3, 'hiddenLayer_Size': 32, 'weightDecay': 0, 'learningRate': 0.001, 'optimizer': 'Nesterov_Accelerated_GD', 'batchSize': 32, 'weightInit': 'xavier', 'activationFunc': 'sigmoid', '_wandb': {}}
2025-03-09 21:59:00,266 INFO    Thread-16 (_run_job):8340 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-09 21:59:00,961 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_finish():2112] finishing run da6401-assignments/assignment1/glm3byz1
2025-03-09 21:59:00,964 ERROR   Thread-16 (_run_job):8340 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-09 21:59:00,964 INFO    Thread-16 (_run_job):8340 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-09 21:59:00,964 INFO    Thread-16 (_run_job):8340 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-09 21:59:00,964 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 1
2025-03-09 21:59:00,964 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_restore():2322] restore
2025-03-09 21:59:00,964 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_restore():2328] restore done
2025-03-09 21:59:02,518 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-09 21:59:02,519 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-09 21:59:02,519 INFO    Thread-16 (_run_job):8340 [wandb_run.py:_footer_sync_info():3917] logging synced files
2025-03-09 21:59:02,524 ERROR   MainThread:8340 [pyagent.py:_run_jobs_from_queue():234] Run glm3byz1 errored:
Traceback (most recent call last):
  File "c:\Python312\Lib\site-packages\wandb\agents\pyagent.py", line 306, in _run_job
    self._function()
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_8340\360757808.py", line 36, in sweep_hyperParameters
    model.train(x_train, y_train, x_val, y_val, num_epochs, learningRate, optimizer, batchSize)
  File "C:\Users\prajv\AppData\Local\Temp\ipykernel_8340\3812825710.py", line 55, in train
    x_train, y_train = x_train[indices], y_train[indices]
                                         ~~~~~~~^^^^^^^^^
IndexError: index 310 is out of bounds for axis 0 with size 10

2025-03-09 21:59:04,534 INFO    MsgRouterThr:8340 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
2025-03-09 21:59:05,485 INFO    Thread-21 (_run_job):8340 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-09 21:59:05,485 INFO    Thread-21 (_run_job):8340 [wandb_setup.py:_flush():67] Configure stats pid to 8340
2025-03-09 21:59:05,485 INFO    Thread-21 (_run_job):8340 [wandb_setup.py:_flush():67] Loading settings from C:\Users\prajv\.config\wandb\settings
2025-03-09 21:59:05,485 INFO    Thread-21 (_run_job):8340 [wandb_setup.py:_flush():67] Loading settings from c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\settings
2025-03-09 21:59:05,490 INFO    Thread-21 (_run_job):8340 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-09 21:59:05,490 INFO    Thread-21 (_run_job):8340 [wandb_init.py:setup_run_log_directory():647] Logging user logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_215905-fwnq7ls3\logs\debug.log
2025-03-09 21:59:05,490 INFO    Thread-21 (_run_job):8340 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to c:\Users\prajv\Downloads\ass1\DA6401_ASSIGNMENT1\wandb\run-20250309_215905-fwnq7ls3\logs\debug-internal.log
2025-03-09 21:59:05,492 INFO    Thread-21 (_run_job):8340 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000020E8B8A3C80>
2025-03-09 21:59:05,492 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():761] calling init triggers
2025-03-09 21:59:05,492 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activationFunc': 'sigmoid', 'batchSize': 32, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 10, 'num_hiddenLayers': 4, 'optimizer': 'nadam', 'weightDecay': 0.0005, 'weightInit': 'xavier'}
config: {'_wandb': {}}
2025-03-09 21:59:05,492 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():784] starting backend
2025-03-09 21:59:05,492 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():788] sending inform_init request
2025-03-09 21:59:05,494 INFO    Thread-21 (_run_job):8340 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-09 21:59:05,494 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():798] backend started and connected
2025-03-09 21:59:05,494 INFO    Thread-21 (_run_job):8340 [wandb_run.py:_config_callback():1261] config_cb None None {'activationFunc': 'sigmoid', 'batchSize': 32, 'hiddenLayer_Size': 128, 'learningRate': 0.001, 'num_epochs': 10, 'num_hiddenLayers': 4, 'optimizer': 'nadam', 'weightDecay': 0.0005, 'weightInit': 'xavier'}
2025-03-09 21:59:05,500 INFO    Thread-21 (_run_job):8340 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-09 21:59:05,527 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():891] updated telemetry
2025-03-09 21:59:05,603 INFO    Thread-21 (_run_job):8340 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
